server/core_dataset1.csv

```diff
@@ -0,0 +1,5 @@
+39965	"Coffee Lake"	10129783825	806937020	123
```

Comment:
These are not used by production, can you move these csv to the test? It would be great when unit tests use them for validation.